src/main/java/nl/topicus/mssql2monetdb/CopyToolConfig.java

```diff
@@ -150,34 +202,74 @@ public CopyToolConfig(String args[]) throws ConfigException, ConfigurationExcept
 			return;
 		}
 
-		configFile = new File(cmd.getOptionValue("config"));
-		
-		LOG.info("Using config file: " + configFile.getAbsolutePath());
-
-		// Load configuration using Commons Configuration lib (allows including other config files)
-		Configuration config = new PropertiesConfiguration(configFile);
-		
-		// replace environment variable references in config and transform into a simple Properties object
-		Properties props = loadEnvironmentVariables(config);
-
-		this.databaseProperties = getAndValidateDatabaseProperties(props);
-		this.sourceDatabases = findSourceDatabases(props);
-		this.tablesToCopy = findTablesToCopy(props);
-		
-		findSchedulerProperties(props);
-			
-		findTriggerProperties(props);
-		
-		this.tempDirectory = findTempDirectory(props);
-
 		this.noSwitch = cmd.hasOption("no-switch");
 		LOG.info("No-Switch-flag set to: " + noSwitch);
 
 		this.switchOnly = cmd.hasOption("switch-only");
 		LOG.info("Switch-Only-flag set to: " + switchOnly);
 		
-		// verify scheduling source
-		//checkSchedulingSource();
+		//manually switch just a single monetdb-table and explicitly require the user to specify only wanting to switch a view to another backing table
+		List<String> requiredOptionsForSchemaSwitchOnly = Arrays.asList("monetdb-table", "monetdb-schema", "monetdb-db", "monetdb-user", "monetdb-password", "monetdb-server", "switch-only");
+		boolean allRequiredOptionsPresent = requiredOptionsForSchemaSwitchOnly
+				.stream()
+				.map(o -> cmd.hasOption(o)
+						&& !options.getOption(o).hasArg() 
+						|| (cmd.getOptionValue(o) != null && !cmd.getOptionValue(o).isEmpty())) 
```

Comment:
Graag verwarring voorkomen wat er eerder wordt geëvalueerd, de AND of de OR, d.m.v. haakjes.

Dus is het:
`cmd.hasOption(o) && (!options.getOption(o).hasArg() || (cmd.getOptionValue(o) != null && !cmd.getOptionValue(o).isEmpty()))`

of 
`
(cmd.hasOption(o) && !options.getOption(o).hasArg()) || (cmd.getOptionValue(o) != null && !cmd.getOptionValue(o).isEmpty())
`